modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/logical/SqlScriptRunner.java

```diff
@@ -595,6 +673,14 @@ private void checkResultsHashed(List<List<?>> res) {
         }
     }
 
+    /** */
+    public static String byteArrayToHex(byte[] arr) {
```

Comment:
IgniteUtils.byteArray2HexString()

Reply:
it converts to upper case, ok i extend utility method, thanks.